website/docs/r/worklink_website_certificate_authority_association.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "aws"
+page_title: "AWS: aws_worklink_website_certificate_authority_association"
+sidebar_current: "docs-aws-resource-worklink-website-certificate-authority-association"
+description: |-
+  Provides a AWS WorkLink Website Certificate Authority Association resource.
+---
+
+# aws_worklink_website_certificate_authority_association
+
+## Example Usage
+
+```hcl
+resource "aws_worklink_fleet" "example" {
+  name  = "terraform-example"
+}
+
+resource "aws_worklink_website_certificate_authority_association" "test" {
+	fleet_arn	= "${aws_worklink_fleet.test.arn}"
+	certificate = "${file("test-fixtures/worklink-website-certificate-authority-association.pem")}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `fleet_arn` - (Required, ForceNew) The ARN of the fleet.
+* `certificate` - (Required, ForceNew) The root certificate of the Certificate Authority.
+* `display_name` - (Optional, ForceNew) The certificate name to display.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `website_ca_id` - A unique identifier for the Certificate Authority.
+
+
+## Import
+
+WorkLink Website Certificate Authority can be imported using `WEBSITE-CA-ID/FLEET-ARN`, e.g.
```

Comment:
Just noting here that when the ID is swapped around, this documentation will also need an update 👍

Reply:
Fixed.